chessington/ui/static/scripts/board.js

```diff
@@ -60,6 +62,7 @@ export class Board {
       
         if (squareData) {
           const piece = await this.buildPiece(square.group, squareData.piece, squareData.player)
+          piece.svg.addClass(this.getChessPositionForRowAndCol(i, j) + '-' + squareData.player + '-' + squareData.piece)
```

Comment:
Should we be setting an `id` over a `class` here? This seems to work for me when using `.attr('id', value)`?
(similarly for when we set the rect attribute above)

Reply:
(Very minor) For consistency it would be nice to have the piece name as lowercase (e.g. atm we have `d2-white-Pawn`):
```suggestion
          const chessPosition = this.getChessPositionForRowAndCol(i, j)
          const pieceId = `${chessPosition}-${squareData.player}-${squareData.piece.toLowerCase()}`
          piece.svg.attr('id', pieceId)
```